src/matcha_ml/core/core.py

```diff
@@ -263,6 +283,27 @@ def provision(
         return matcha_state_service.fetch_resources_from_state_file()
 
 
-def stack_set(stack: str) -> str:
-    """Placeholder for core matcha stack set functionality."""
-    return stack
+def stack_set(stack_name: str) -> None:
```

Comment:
We should include a check here to prevent setting the stack when resources are provisioned